README.md

```diff
@@ -371,22 +409,14 @@ $ curl -v --cacert certs/CA.crt  \
           --key certs/BOOTSTRAP.key  \
           -H 'Content-Type: application/cbor' \
           -H 'Accept: application/cbor' \
-          https://MBP2021.local:1443/api/v1/ccs
-```
-
-#### Response
-
-Replies with a CBOR array containing the following fields:
-
-```cddl
-{
-   1 => tstr,  ; Hubname
-   2 => int,   ; Port
-}
```

Comment:
This should stay under `####` and is still valid as the CBOR response.

Reply:
When we run the curl command for ccs still we get a response in binary output

```
 curl -v --cacert certs/CA.crt  \
          --cert certs/BOOTSTRAP.crt \
          --key certs/BOOTSTRAP.key  \
          -H 'Content-Type: application/cbor' \
          -H 'Accept: application/cbor' \
          https://arm-virtual-machine.local:1443/api/v1/ccs
```

Response on the console:

```
< content-type: application/cbor
< content-length: 12
< date: Thu, 06 Oct 2022 20:07:03 GMT
< 
* TLSv1.2 (IN), TLS header, Supplemental data (23):
Warning: Binary output can mess up your terminal. Use "--output -" to tell 
Warning: curl to output it to your terminal anyway, or consider "--output 
Warning: <FILE>" to save to a file.
* Failure writing output to destination
* stopped the pause stream!
* Connection #0 to host arm-virtual-machine.local left intact
```

So I removed the response block for cbor

Reply:
I think showing the CDDL output like this is useful to indicate to people what format that binary data consist of, so that they know how to parse it. It's hard to display the CBOR another way.

Reply:
we are getting CBOR encoded in binary format and unreadable.
Even utility `cbor_decoder.go` decode and print the response in `JSON` format and it is not CDDL and it confuses the reader/user, I am not getting why we need to show CDDL for only ccs and not for other rest APIs CBOR response

Reply:
We should document these consistently everywhere. Current docs are inconsistent, I agree. I'll do that myself in a new PR, though, not to block this one.